Arrowgene.Ddon.GameServer/Handler/WarpGetReturnLocationHandler.cs

```diff
@@ -1,28 +1,109 @@
 using Arrowgene.Ddon.Server;
-using Arrowgene.Ddon.Server.Network;
 using Arrowgene.Ddon.Shared.Entity.PacketStructure;
 using Arrowgene.Ddon.Shared.Model;
-using Arrowgene.Ddon.Shared.Network;
 using Arrowgene.Logging;
+using System.Collections.Generic;
 
 namespace Arrowgene.Ddon.GameServer.Handler
 {
-    public class WarpGetReturnLocationHandler : StructurePacketHandler<GameClient, C2SWarpGetReturnLocationReq>
+    public class WarpGetReturnLocationHandler : GameRequestPacketHandler<C2SWarpGetReturnLocationReq, S2CWarpGetReturnLocationRes>
     {
         private static readonly ServerLogger Logger = LogProvider.Logger<ServerLogger>(typeof(WarpGetReturnLocationHandler));
 
+        /// <summary>
+        /// What startpos to use for each map. This is usually in front of a riftstone.
+        /// </summary>
+        private static readonly Dictionary<uint, uint> RespawnStartPosMap = new()
+        {
+            { 2, 0}, // White Dragon Temple
```

Comment:
People will be unhappy about this because the instance will reset 🤣

Reply:
Im sure its supposed to, if it didnt up until now that was an oversight

Reply:
We used to send people back to the audience chamber which is not considered a safe area, so there was no instance reset.